mod/InGameTracker/TrackerManager.cs

```diff
@@ -54,6 +61,8 @@ private void Awake()
             }
 
             inventoryMode = gameObject.AddComponent<TrackerInventoryMode>();
+            checklistMode = gameObject.AddComponent <TrackerItemChecklistMode>();
```

Comment:
Doesn't this line add the unfinished tracker mode in-game? I'm fine with committing a bunch of dead code inside `InGameTracker/` if that makes development easier, but it's gotta be actually dead :) (or finished)

Less importantly: _Item_ checklist? Not location?

Reply:
It adds an empty component that will be referenced later, this won't actually have any effect (but I can comment it out if you'd prefer)
And yes location checklist